db/pagination/pagination.go

```diff
@@ -37,40 +37,82 @@ const (
 type Paginator struct {
 	Key    string
 	Order  string
-	Limit  uint64
+	Limit  int64
```

Comment:
could we come back to unsigned? semantically, the limit could not be negative

Reply:
working on it, not trivial as uint64 max value won't work with js...